.github/workflows/daily-deploy-production.yml

```diff
@@ -11,7 +11,7 @@ on:
         description: Deploy specific commit
         required: false
   schedule:
-    - cron: 0 18 * * 1-5
+    - cron: 0 19 * * 1-5
```

Comment:
Remember to update related documentation if/when this gets merged

Reply:
The [deployments](https://depo-platform-documentation.scrollhelp.site/developer-docs/deployments) documentation has been updated in confluence ✅. The changes should be published today at 3 PM ET